apps/api/src/app/layouts/usecases/create-default-layout-change/create-default-layout-change.usecase.ts

```diff
@@ -21,7 +21,7 @@ export class CreateDefaultLayoutChangeUseCase {
   ) {}
 
   async execute(command: CreateDefaultLayoutChangeCommand): Promise<void> {
-    let item: LayoutEntity | LayoutDto = await this.layoutRepository.findOne({
+    let item: LayoutEntity | LayoutDto | null = await this.layoutRepository.findOne({
```

Comment:
It doesn't feel right this typing. LayoutEntity and LayoutDto shouldn't be used in the same place even if then they are interchangeable because same structure. I know it is inherited though.

Reply:
I don't know "why's" here, but I've fixed just the type here. We do assign different values to the `item` variable.